packages/rlc-common/src/buildIndexFile.ts

```diff
@@ -327,8 +331,35 @@ function generateRLCIndex(file: SourceFile, model: RLCModel) {
     ]);
   }
 
+  reExportFileHelperFromCore(file, model);
+
   file.addExportAssignment({
     expression: createClientFuncName,
     isExportEquals: false
   });
 }
+
+function reExportFileHelperFromCore(file: SourceFile, model: RLCModel) {
+  // TODO: remember to remove this when non-branded core has exported file helper
```

Comment:
Remember to comment this out when the core is ready - https://github.com/Azure/autorest.typescript/issues/2299.